pkg/replication/replication.go

```diff
@@ -59,6 +64,7 @@ type Options struct {
 	IsSCSet                bool
 	ReplicateDeletes       string // replicate versioned deletes
 	ReplicateDeleteMarkers string // replicate soft deletes
+	ReplicateSync          string // whether synchronous replication is specified.
```

Comment:
why not make this `boolean` to start with?  - you don't need to bring in `mc` logic here. `mc` can still support words like `sync` and `async` but API wise this doesn't need to be `string`

Reply:
this is needed for `mc replicate edit` option to make out whether user is opting for synchronous/asynchronous replication. It would not be possible to make out with a boolean state if user is changing it or left it unset